src/mono/wasm/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs

```diff
@@ -884,5 +884,46 @@ public static unsafe int Main(string[] argv)
         [BuildAndRun(host: RunHost.Chrome, aot: false)]
         public void EnsureWasmAbiRulesAreFollowedInInterpreter(BuildArgs buildArgs, RunHost host, string id) =>
             EnsureWasmAbiRulesAreFollowed(buildArgs, host, id);
+
+        [Theory]
+        [BuildAndRun(host: RunHost.Chrome, aot: false)]
+        public void UCOWithSpecialCharacters(BuildArgs buildArgs, RunHost host, string id)
+        {
+            string cCode =
```

Comment:
Please, move the C code to a file in `Wasm.Buid.Tests.Programs` and copy it from there when instantiating the test app

Reply:
Maybe it will be better to move C code to https://github.com/dotnet/runtime/tree/main/src/mono/wasm/testassets/native-libs ?  `Wasm.Buid.Tests.Programs` isn't for Program.cs ?

Reply:
Ook, let's put it beside other native files